src/nvim/ex_getln.c

```diff
@@ -190,11 +128,41 @@ typedef struct command_line_state {
   long *b_im_ptr;
 } CommandLineState;
 
+// Variables shared between getcmdline(), redrawcmdline() and others.
```

Comment:
The diffs are messy here and introduce unnecessary changes. Can you make sure the changes introduced are minimal?